packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -95,7 +99,13 @@ export class PluginPathsServiceImpl implements PluginPathsService {
      * Generate time folder name in format: YYYYMMDDTHHMMSS, for example: 20181205T093828
      */
     private gererateTimeFolderName(): string {
```

Comment:
[minor] since you are already updating the following method, do you mind renaming the method to fix the typo:
```ts
generateTimeFolderName()
```

Reply:
done